Index: src/com/dmdev/cs/homework/TimeZone.java
===================================================================
diff --git a/src/com/dmdev/cs/homework/TimeZone.java b/src/com/dmdev/cs/homework/TimeZone.java
new file mode 100644
--- /dev/null	
+++ b/src/com/dmdev/cs/homework/TimeZone.java	
@@ -0,0 +1,8 @@
+package com.dmdev.cs.homework;
+
+public class TimeZone {
+
+    public int getTimeZone(int minutes) {
+
+    }
+}
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	
+++ b/.idea/misc.xml	
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="corretto-11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="corretto-17" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
 </project>
\ No newline at end of file
